catalog-rest-service/src/main/resources/json/schema/entity/services/connections/database/clickhouseConnection.json

```diff
@@ -48,9 +48,9 @@
       "description": "Host and port of the Clickhouse service.",
       "type": "string"
     },
-    "database": {
-      "title": "Database",
-      "description": "Database of the data source. This is optional parameter, if you would like to restrict the metadata reading to a single database. When left blank, OpenMetadata Ingestion attempts to scan all the databases.",
+    "databaseSchema": {
```

Comment:
clickhouse does not support database. Fixed the schema

Reply:
@pmbrull we are still asking redshift for database as required input on the UI? any reason for that

Reply:
we need the database to perform the initial connection. Updated the schema and code to handle multiple databases if needed